Description: default the value of EDITOR with Debian's sensible-editor instead of vi
 the original bug that triggered this change was that quilt doesn't honour $VISUAL
Bug-Debian: http://bugs.debian.org/509076
Forwarded: not-needed
Author: Ryan Niebur <ryanryan52@gmail.com>

---
 quilt/edit.in   |    2 +-
 quilt/header.in |    2 +-
 quilt/mail.in   |    2 +-
 test/edit.test  |    1 +
 4 files changed, 4 insertions(+), 3 deletions(-)

Index: b/quilt/edit.in
===================================================================
--- a/quilt/edit.in
+++ b/quilt/edit.in
@@ -6,7 +6,7 @@
 #
 #  See the COPYING and AUTHORS files for more details.
 
-: ${EDITOR:=vi}
+: ${EDITOR:=sensible-editor}
 
 # Read in library functions
 if [ "$(type -t patch_file_name)" != function ]
Index: b/quilt/header.in
===================================================================
--- a/quilt/header.in
+++ b/quilt/header.in
@@ -17,7 +17,7 @@
 	. $QUILT_DIR/scripts/patchfns
 fi
 
-: ${EDITOR:=vi}
+: ${EDITOR:=sensible-editor}
 
 usage()
 {
Index: b/quilt/mail.in
===================================================================
--- a/quilt/mail.in
+++ b/quilt/mail.in
@@ -6,7 +6,7 @@
 #
 #  See the COPYING and AUTHORS files for more details.
 
-: ${EDITOR:=vi}
+: ${EDITOR:=sensible-editor}
 
 # Read in library functions
 if [ "$(type -t patch_file_name)" != function ]
Index: b/test/edit.test
===================================================================
--- a/test/edit.test
+++ b/test/edit.test
@@ -6,6 +6,7 @@
 < sed -e 's:foo:bar:' $1 > $1.new
 < mv $1.new $1
 $ chmod +x editor
+$ unset VISUAL
 $ export EDITOR=%{PWD}/editor
 
 $ quilt new patch
